strawberry/types/types.py

```diff
@@ -120,6 +125,8 @@ def get_name_from_type(self, type_: Union[StrawberryType, type]) -> str:
             return type_.name
         elif isinstance(type_, StrawberryUnion):
             return type_.name
+        elif isinstance(type_, StrawberryList):
+            return self.get_name_from_type(type_.of_type) + "List"
```

Comment:
I wonder if there are other `StrawberryContainer` subclasses that require a similiar treatment.

Maybe it would be better to add some kind of `name` property/function to the container type itself so it is in one place and not scattered over the code base?

Reply:
that’s a good point, for now I think we only have List and Optional as containers :)

Can you add that too? ^^

Reply:
Done. :+1: